permissions_validators/src/private_blockchain/query.rs

```diff
@@ -30,7 +30,26 @@ impl<W: WorldTrait> IsAllowed<W, QueryBox> for OnlyAccountsDomain {
                 Err("Only access to the domain of the account is permitted.".to_owned())
             }
             FindAllRoles(_) => Ok(()),
+            FindAllRoleIds(_) => Ok(()),
+            FindRoleByRoleId(_) => Ok(()),
             FindAllPeers(_) => Ok(()),
+            FindAllActiveTriggerIds(_) => Ok(()),
+            FindTriggerById(_) => Ok(()), // TODO: should we allow people to get any trigger they like in a private blockchain also?
+            FindTriggerKeyValueByIdAndKey(query) => {
```

Comment:
I think all these are world-level access and should return `Err` as `OnlyAccountsDomain` means a restriction to domain-level access